CodeToReview.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+
+
+   O | O | O
+  -----------
+   O | O | O
+  -----------
+   O | O | O
+
+
+ */
+package tictactoe;
+
+import java.util.Scanner;
+
+public class TicTacToe {
+
+    /**
+     * @param args the command line arguments
+     * @throws java.lang.InterruptedException
+     */
+    public static void main(String[] args) throws InterruptedException {
+        Scanner input = new Scanner(System.in);
+        String a1 = "1";
```

Comment:
The variables can be renamed to increase the readability which will help understanding the exact usage of the variable